README.md

```diff
@@ -9,6 +9,20 @@
 
 This is the Android client for the Mentorship System.
 
+## Features
+
+* A member gets an option to be a mentor, mentee or both.
+* Building an engaging profile -                        
+ A member can customise the app profile with username, bio, skills, interests, location, occupation which can be updated anytime by a member. 
+* A member can view other members of the application or search for them in the Members Page.
+* A member can send mentorship requests to other members and accept or reject mentorship requests from other members.
+* A member can track the number of Pending Resquests, Accepted Requests, Rejected Requests, and Completed Requests and view Recent Achievements on the Home Page.  
+* A member can view  details of pending, past, all the mentorship reqests in the Requests Page.
+* A member can create, update, or delete tasks in the current mentorship relation.
+* A member can give feedback as a bug, suggestion, other response to the organisation.
```

Comment:
A member can send feedback to the Mentorship System such as reporting a bug, giving suggestions or other comments.